config/bidderinfo_test.go

```diff
@@ -184,6 +160,161 @@ func TestLoadBidderInfo(t *testing.T) {
 	}
 }
 
+func TestSyncerOverride(t *testing.T) {
```

Comment:
It doesn't look like you have any tests where you confirm that some fields are untouched while others are overridden in one request. (Where the override syncer only provides a subset of original fields.)

Reply:
Good point. I intend to write this test soon.